index.html

```diff
@@ -129,7 +129,7 @@ <h2 id="project_tagline">A framework to secure the integrity of software supply
              in-toto is maintained by <a
              href="https://github.com/in-toto/">The in-toto
              team</a>, a group of researchers and developers from <a
-             href="engineering.nyu.edu">New York University</a> and <a
+             href="http://engineering.nyu.edu">New York University</a> and <a
```

Comment:
@SantiagoTorres I checked that. https is redirecting to http for that url so no point.